loopFibonacci.c

```diff
@@ -0,0 +1,21 @@
+#include<stdio.h>
```

Comment:
Did you get to run your code? you had simple syntax errors, but once I fixed these, your code ran perfectly!